### PR TITLE
Acxiom Data Enhancement Export

### PR DESCRIPTION
**Pull request by varholik**

### CODE REVIEW

File: help/destinations/catalog/advertising/acxiom-data-activation.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+---
+# [!DNL Acxiom Data Activation] destination connection
+
+>[!NOTE]
+>
+>The [!DNL Acxiom Data Activation] destination is in beta.  This destination connector and documentation page are created and maintained by the Acxiom team. For any inquiries or update requests, please contact them directly at acxiom-adobe-help@acxiom.com.
+
+## Overview {#overview}
+
+Use Acxiom Data Activation connector to activate first-party profiles in Real-Time CDP to Acxiom for enrichment and use across marketing channels.
```
Comment (vgiurgiu) [inline help/destinations/catalog/advertising/acxiom-data-activation.md]: minor - use _the_ Acxiom Data Activation connector

File: help/destinations/catalog/advertising/acxiom-data-activation.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
```
Comment (vgiurgiu) [inline help/destinations/catalog/advertising/acxiom-data-activation.md]: can we also add a paragraph at the top of the page explaining how the purpose of this connector is different from the other one by Acxiom - Prospect Suppression?

File: help/destinations/catalog/advertising/acxiom-data-activation.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+![The destination catalog with the Acxiom destination selected.](../../assets/catalog/advertising/acxiom/image-destination-enhancement-catalog.png)
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the Acxiom Data Activation destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+### Use case 1: Enhance Customer Datasets {#use-case-1}
+
+Marketing professionals aiming to enhance the effectiveness of their outreach strategies by adding selected elements to better target campaigns. This strategic approach helps refine the audience and contributes to a more focused and efficient marketing effort.
```
Comment (vgiurgiu) [inline help/destinations/catalog/advertising/acxiom-data-activation.md]: "Marketing professionals aiming to enhance the effectiveness of their outreach strategies by adding selected elements to better target campaigns." -> this sentence is not grammatically correct. can you please reword ?

also, can we please explain in simpler terms what this connector does? how does this aid customers in data enrichment>

File: help/destinations/catalog/advertising/acxiom-data-activation.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+## Prerequisites {#prerequisites}
+
```
Comment (vgiurgiu) [inline help/destinations/catalog/advertising/acxiom-data-activation.md]: Are there any prerequisites on the Acxiom side?

File: help/destinations/catalog/advertising/acxiom-data-activation.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate audience data to batch profile export destinations](/help/destinations/ui/activate-batch-profile-destinations.md) for instructions on activating audiences to this destination.
+
+### Mapping suggestions
+
+Processing requires name and address elements, while not all elements are required providing as much as possible will aid in successful matching.  Mapping suggestions are provided in the table below listing attributes on your destination side that are used by Acxiom processing that customers can map profile attributes to.  This should be treated as suggestions as not all elements are required and the source values will depend on the needs of the account.
```
Comment (vgiurgiu) [inline help/destinations/catalog/advertising/acxiom-data-activation.md]: some wording suggestions here:

The correct processing of files on the Acxiom side requires name and address elements. While not all elements are required, providing as much as possible will aid in successful matching.  

Mapping suggestions are provided in the table below listing attributes on your destination side that are used by Acxiom processing that customers can map profile attributes to. Treat these elements as suggestions as not all elements are required and the source values will depend on the needs of the account.

______

One comment though - if any of the elements are required, can we clearly point that out please in the table?

File: help/destinations/catalog/advertising/acxiom-data-activation.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+| lastName     | The person.name.lastName value in Experience Platform.      |
+| address1     | The mailingAddress.street1 value in Experience Platform.    |
+| address2     | The mailingAddress.street2 value in Experience Platform.    |
+| city         | The mailingAddress.city value in Experience Platform.       |
+| state        | The mailingAddress.state value in Experience Platform.      |
+| zip          | The mailingAddress.postalCode value in Experience Platform. |
+
+>[!NOTE]
+>
+>Additional fields not listed above will be included on the export, but will be ignored by Acxiom processing.
```
Comment (vgiurgiu) [inline help/destinations/catalog/advertising/acxiom-data-activation.md]: minor rewording suggestion - 

If you map additional fields not listed above in the dataflow, these will be included in the data export, but will be ignored by Acxiom processing.

File: help/destinations/catalog/advertising/acxiom-data-activation.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+
+## Review your dataflow
+
+Use the review page for a summary of your dataflow prior to submission
```
Comment (vgiurgiu) [inline help/destinations/catalog/advertising/acxiom-data-activation.md]: not sure if we need lines https://github.com/AdobeDocs/experience-platform.en/pull/323/files#diff-15b5cff9d1a57cb2fdfb1e362eeb4c11828036e2acad8c78856afc3b85393b6dR141-R143 

The review step is described in the tutorial that we linked to in line 120. We should call out the review step specifically only if there's anything particular that you want to point out in it.

File: help/destinations/catalog/advertising/acxiom-data-activation.md [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+
+![Review](../../assets/catalog/advertising/acxiom/image-destination-enhancement-review.png)
+
+## Next steps
+
+By following this tutorial, you have successfully created a dataflow to export batch data from Experience Platform to your [!DNL Acxiom] managed S3 location. You would need to contact your Acxiom representative with the name of the account, filename, and the bucket path so that processing can set up.
```
Comment (vgiurgiu) [inline help/destinations/catalog/advertising/acxiom-data-activation.md]: minor: 

By following this tutorial, you have successfully created a dataflow to export profile data from Experience Platform to your [!DNL Acxiom] managed S3 location. Next, you need to contact your Acxiom representative with the name of the account, file names, and the bucket path so that processing can set up.

File: help/destinations/catalog/advertising/acxiom-data-activation.md [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, read the [Data Governance overview](/help/data-governance/home.md).
+
+
+## Exported data / Validate data export {#exported-data}
```
Comment (vgiurgiu) [inline help/destinations/catalog/advertising/acxiom-data-activation.md]: let's just leave `## Validate data export {#exported-data}`

File: help/destinations/catalog/advertising/acxiom-data-enhancement-export.md [hunk 10]
```diff
@@ -0,0 +1,163 @@
+---
```
Comment (vgiurgiu) [inline help/destinations/catalog/advertising/acxiom-data-enhancement-export.md]: this file seems like a duplicate of the file acxion-data-activation.md above. Are you planning to keep only one of them?

File: help/destinations/catalog/advertising/acxiom-prospect-suppression.md [hunk 11]
```diff
@@ -0,0 +1,163 @@
+---
+title: Acxiom Prospect-Suppression
```
Comment (vgiurgiu) [inline help/destinations/catalog/advertising/acxiom-prospect-suppression.md:2]: what are the plans with this destination document? are you planning to have documentation for both Acxiom destinations in the same PR? 

I know that we were discussing the Prospect Suppression destination documentation in the separate PR https://github.com/AdobeDocs/experience-platform.en/pull/312

**Review by vgiurgiu — COMMENTED** [review]
hi @varholik thanks for submitting the documentation for the Data Activation destination.  please see my comments and let me know if you have any questions. Thanks!

File: help/destinations/catalog/advertising/acxiom-data-enhancement-export.md (hunk 10 above)
Comment (varholik) [reply help/destinations/catalog/advertising/acxiom-data-enhancement-export.md]: No, this file should have been replaced then the card was renamed.  I'll delete that file.  Thanks

File: help/destinations/catalog/advertising/acxiom-prospect-suppression.md (hunk 11 above)
Comment (varholik) [reply help/destinations/catalog/advertising/acxiom-prospect-suppression.md:2]: This is a second destination card.  This documentation should be kept independent of the prospect.  While we would like to have both completed, completing prospect would come first, as I think they want that card ready for the summit.

File: help/destinations/catalog/advertising/acxiom-data-activation.md (hunk 5 above)
Comment (varholik) [reply help/destinations/catalog/advertising/acxiom-data-activation.md]: I don't think any fields are specifically required, but the lack of too many of elements of it might prevent any successful matches.

File: help/destinations/catalog/advertising/acxiom-data-activation.md (hunk 4 above)
Comment (varholik) [reply help/destinations/catalog/advertising/acxiom-data-activation.md]: Not really, but as alluded in the Next Steps section, there would need to be a discussion with the contact within Acxiom, they would need to prepare to consume these files and would likely need to discuss the data elements ordered.   Some of that conversation might be better before dataflow are prepared, but I am not sure I would call that a hard prerequisite.

File: help/destinations/catalog/advertising/acxiom-data-activation.md (hunk 9 above)
Comment (varholik) [reply help/destinations/catalog/advertising/acxiom-data-activation.md]: Removed the "Exported data /" prefix

File: help/destinations/catalog/advertising/acxiom-data-activation.md (hunk 8 above)
Comment (varholik) [reply help/destinations/catalog/advertising/acxiom-data-activation.md]: Changed to use suggested wording.

File: help/destinations/catalog/advertising/acxiom-data-activation.md (hunk 7 above)
Comment (varholik) [reply help/destinations/catalog/advertising/acxiom-data-activation.md]: The review section has been removed from this document.

File: help/destinations/catalog/advertising/acxiom-data-activation.md (hunk 6 above)
Comment (varholik) [reply help/destinations/catalog/advertising/acxiom-data-activation.md]: Changed to use suggested wording.

File: help/destinations/catalog/advertising/acxiom-data-activation.md (hunk 1 above)
Comment (varholik) [reply help/destinations/catalog/advertising/acxiom-data-activation.md]: Changed to use suggested wording.

File: help/destinations/catalog/advertising/acxiom-data-activation.md (hunk 5 above)
Comment (varholik) [reply help/destinations/catalog/advertising/acxiom-data-activation.md]: Used suggested wording

File: help/destinations/catalog/advertising/acxiom-data-activation.md (hunk 2 above)
Comment (varholik) [reply help/destinations/catalog/advertising/acxiom-data-activation.md]: Updated overview to try to clarify the purpose

File: help/destinations/catalog/advertising/acxiom-data-activation.md (hunk 3 above)
Comment (varholik) [reply help/destinations/catalog/advertising/acxiom-data-activation.md]: Tried to clarify use case.
